src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQContext.cpp

```diff
@@ -20,13 +20,30 @@
 #include <opm/parser/eclipse/EclipseState/Schedule/TimeMap.hpp>
 #include <opm/parser/eclipse/EclipseState/Schedule/SummaryState.hpp>
 #include <opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQContext.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQState.hpp>
 
 
 namespace Opm {
 
-    UDQContext::UDQContext(const UDQFunctionTable& udqft_arg, const SummaryState& summary_state_arg) :
+namespace {
+
+bool is_udq(const std::string& key) {
+    if (key.size() < 2)
+        return false;
```

Comment:
This should **probably** be `key.size() < 3`.  I don't think we want to support variables named `WU`.

Reply:
No - that is probably right; but that should be handled in the general code for summary keys - as I see it.